eng/common/scripts/Prepare-Release.ps1

```diff
@@ -64,6 +64,55 @@ function Get-ReleaseDay($baseDate)
   return $baseDate;
 }
 
+function Check-ApiReviewStatus($packageName, $language)
+{
+  # Get API view URL and API Key to check status
+  Login-AzAccount
+  $url = Get-AzKeyvaultSecret -VaultName "AzureSDKPrepRelease-KV" -Name APIURL -AsPlainText
+  $apiKey = Get-AzKeyvaultSecret -VaultName "AzureSDKPrepRelease-KV" -Name APIKEY -AsPlainText
+
+  $lang = $language
+  # Update language name to match those in API cosmos DB. Cosmos SQL is case sensitive and handling this within the query makes it slow.
+  if($lang -eq 'javascript'){
+    $lang = "JavaScript"
+  }
+  elseif ($lang -eq "dotnet"){
+    $lang = "C#"
+  }
+  elseif ($lang -eq "java"){
+    $lang = "Java"
+  }
+  elseif ($lang -eq "python"){
+    $lang = "Python"
+  }
+
+  $headers = @{ "ApiKey" = $apiKey }
+  $body = @{
+    language = $lang
+    packageName = $packageName
+  }
+
+  try
+  {
+    $Response = Invoke-WebRequest $url -Method 'GET' -Headers $headers -Body $body
+    Write-Host "Response: $($Response.Content)"
+    if ($Response.StatusCode -eq '200')
+    {
+      Write-Host "API Review is approved for package $($packageName)"
+    }
+    else
+    {
+      Write-Error "API Review is not approved for package $($packageName). Please get automatic API review approved by architects."
```

Comment:
Can you dump out the URL to the API review here too? Just for convenience?

Reply:
Sure. I will need a change in APIView system itself for this change. I will include it in separate PR once that change is completed.